components/notebook-controller/config/overlays/openshift/params.env

```diff
@@ -0,0 +1 @@
+odh-kf-notebook-controller-image=quay.io/opendatahub/kubeflow-notebook-controller:1.7-35b81f5
```

Comment:
So when we previously referenced the image as

```
image: quay.io/opendatahub/odh-notebook-controller:latest
```

that meant that whatever version of odh user installed, they would get the latest controller image we built?

does the odh operator uses some mechanism to set operator image hashes? the rhoai operator certainly does something like that

Reply:
> image: quay.io/opendatahub/odh-notebook-controller:latest

I didn't check the code thoroughly, but you mean, we referenced also an image from a different repository before?

Reply:
This is just the https://issues.redhat.com/browse/RHOAIENG-9234 problem, nothing new.

Reply:
>that meant that whatever version of odh user installed, they would get the latest controller image we built?
`Latest` was the fall back, it like if kustomize fails to patch the image, it would just stay `latest`.
in previous version, kustomize image was been used.

>does the odh operator uses some mechanism to set operator image hashes? the rhoai operator certainly does something like that

rhoai operator switch image based on the sha give from builds.
perhaps worth check with operator team.